src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.0.1")]
+[assembly: AssemblyFileVersion("2.0.1")]
+[assembly: AssemblyInformationalVersion("2.0.1")]

```

Comment:
@adamralph, I'm not so sure about changing the version number here. In general, I'd be inclined to do it in a separate PR (I know, the commit is separate, and that's good, but it still feels a little weird to me. Any reason you didn't use `rake set_version[new_version]` to set the version? Not convenient enough?)

(While we're on the subject, I cannot find the 2.0.1 release. Am I blind? Stupid?
I can find the milestone and release checklist issue. Did `rake create_milestone` you?)

All this may be academic, as I've merged #736, so we have a new feature - Source stepping via GitLink, and we're probably better off releasing 2.1.0 as you suggested.